src/components/LabeledDropdown.module.css

```diff
@@ -0,0 +1,44 @@
+.labeledDropdown {
+  display: flex;
+  flex-direction: row;
+  gap: 6px;
+  align-items: center;
+  width: max-content;
+}
```

Comment:
Fixed! I'm keeping the display mode as flex, however, since otherwise the align-items property doesn't work correctly and the elements jump around on load.